crates/subspace-fraud-proof/src/lib.rs

```diff
@@ -190,7 +191,7 @@ impl<Block, C> FetchRuntimeCode for RuntimCodeFetcher<Block, C>
 where
     Block: BlockT,
     C: ProvideRuntimeApi<Block> + Send + Sync,
-    C::Api: ExecutorApi<Block>,
+    C::Api: ExecutorApi<Block, SecondaryHash>,
```

Comment:
Hm... I though we planned to have generics for these instead of fixed types.

Reply:
I was too lazy, fixed :P